.travis.yml

```diff
@@ -8,9 +8,7 @@ env: TEST_SUITE=test-main
 matrix:
   include:
   - python: "3.6"
-    env: TEST_SUITE=lint
-  - python: "3.6"
-    env: TEST_SUITE=run-mypy
+    env: TEST_SUITE=test-static
```

Comment:
maybe `TEST_SUITE=static-analysis`?  I feel like we probably want this to be clearer than "test-static" as to what it is.